calico/getting-started/kubernetes/windows-calico/quickstart.mdx

```diff
@@ -475,19 +486,18 @@ c:\k\kubeadm.exe join 172.16.101.139:6443 --token v8w2jt.jmc45acn85dbll1e --disc
 - Run the join command on the Windows node. Shortly after it completes successfully, the Windows node will appear in `kubectl get nodes`.
   The new node's status will be NotReady since the Calico CNI has not yet been installed.
 
-#### Migrating from {{prodnameWindows}} installed manually
+#### Migrating from {{prodnameWindows}} installed manually to an HPC installation
 
 If your Windows nodes already have {{prodnameWindows}} installed using the manual installation method, you can continue this quickstart guide
```

Comment:
```suggestion
The HPC installation in this guide is operator-based and will automatically migrate your Windows nodes that have {{prodnameWindows}} installed using the manual installation method. This installation will uninstall any existing {{prodnameWindows}} services, and overwrite the {{prodnameWindows}} installation files with those included in the `calico/cni-windows` and `calico/node-windows` images. If `kubelet` was installed using `{{rootDirWindows}}\kubernetes\install-kube-services.ps1`, this service will remain installed. If the `kube-proxy` service was also installed using `{{rootDirWindows}}\kubernetes\install-kube-services.ps1`, it will be stopped and uninstalled by the `calico-node-windows` HPC initContainers. It is recommended to run `kube-proxy-windows` as hostprocess container images using the [manifest provided in the kubernetes-sigs sig-windows-tools repository](https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/hostprocess/calico/kube-proxy/kube-proxy.yml).
```

Reply:
Done, thanks!